.github/workflows/build.yml

```diff
@@ -0,0 +1,28 @@
+# This workflow will build a .NET project
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-net
+
+name: .NET
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build:
```

Comment:
Trivial nit-pick / minor (non-blocking) refinement suggestion: the job builds and tests, so perhaps update the job ID and name to reflect the extra bits it's doing?

```yml

jobs:
  build_and_test:
    name: Build and test
```
 
Not worth blocking the PR over, of course! :)

See also: 
- https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_id
- https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idname


![image](https://github.com/DFE-Digital/search-prototype/assets/96429508/cacc001d-c229-4c69-8fbc-759464d1bce4)

Reply:
yes, I can do - it came from github out-of-the-box, so I followed it as is